### ISSUE TITLE
Node version requirements in docs seem to be incorrect

### ISSUE THREAD

**Issue by thor84no** [nodefluent/kafka-streams#38]
The docs say the requirement is:

> nodejs should be version >= 6.10, suggested: >= 8.6.x

But the dependency on sinek appears to require async-await which requires at least node 7.6. Is there a work around that I'm missing?

**Comment by thor84no** [comment]
I've found a work-around. It's happening because your version isn't locked down in package.json; it's "^6.2.2" and unfortunately sinek has added the async-await as a minor version change. I found a work around that works by adding the following to my package.json:

```
  "resolutions": {
    "kafka-streams/sinek": "6.2.2"
  }
```

Unfortunately this only seems to work using `yarn`. I still haven't found an equivalent for `npm`.
